[invalid.py]
  [test_params_descriptor_invalid_type[False\]]
    expected: FAIL

  [test_params_descriptor_invalid_type[SOME_STRING\]]
    expected: FAIL

  [test_params_descriptor_invalid_type[42\]]
    expected: FAIL

  [test_params_descriptor_invalid_type[descriptor3\]]
    expected: FAIL

  [test_params_descriptor_invalid_type[descriptor4\]]
    expected: FAIL

  [test_params_descriptor_invalid_type[descriptor5\]]
    expected: FAIL

  [test_params_descriptor_invalid_type[None\]]
    expected: FAIL

  [test_params_descriptor_invalid_type[descriptor7\]]
    expected: FAIL

  [test_params_descriptor_invalid_value[descriptor0\]]
    expected: FAIL

  [test_params_state_invalid_type[False\]]
    expected: FAIL

  [test_params_state_invalid_type[42\]]
    expected: FAIL

  [test_params_state_invalid_type[state2\]]
    expected: FAIL

  [test_params_state_invalid_type[state3\]]
    expected: FAIL

  [test_params_state_invalid_type[None\]]
    expected: FAIL

  [test_params_state_invalid_type[state5\]]
    expected: FAIL

  [test_params_state_invalid_value[UNKOWN\]]
    expected: FAIL

  [test_params_state_invalid_value[Granted\]]
    expected: FAIL

  [test_params_origin_invalid_type[False\]]
    expected: FAIL

  [test_params_origin_invalid_type[42\]]
    expected: FAIL

  [test_params_origin_invalid_type[origin2\]]
    expected: FAIL

  [test_params_origin_invalid_type[origin3\]]
    expected: FAIL

  [test_params_origin_invalid_type[None\]]
    expected: FAIL

  [test_params_origin_invalid_type[origin5\]]
    expected: FAIL
